common/Helpers/ManagementInfrastructureHelper.cs

```diff
@@ -0,0 +1,67 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using System.Globalization;
+using Microsoft.Management.Infrastructure;
+using Serilog;
+
+namespace DevHome.Common.Helpers;
+
+// States based on InstallState value in Win32_OptionalFeature
+// See: https://learn.microsoft.com/windows/win32/cimwin32prov/win32-optionalfeature
+public enum FeatureAvailabilityKind
```

Comment:
This is duplicated.  Consider moving the enum to a common location.

Reply:
coping the comment I told replied to Vineeth with:

> This is inside the HyperV extension project, while the other is inside DevHome.Common. I didn't want to reference Dev Home from the HyperV extension and reference code from the HyperV extension in Dev Home itself. So I decided duplication was the better way to go about this to keep the two separate